.github/workflows/pr_ok_to_test.yaml

```diff
@@ -0,0 +1,256 @@
+name: PR
```

Comment:
choose unique name
PR_Verified

same as the name of the file
pr_verified.yaml

Reply:
done